runtime/src/main/java/org/corfudb/runtime/Restore.java

```diff
@@ -16,6 +16,8 @@
 import org.corfudb.runtime.view.TableRegistry;
 import org.corfudb.util.serializer.Serializers;
 
+import javax.annotation.Nonnull;
```

Comment:
Should it be lombok?

Reply:
We have these similar annotations from both lombok and javax in Corfu. Are you suggesting that we should prefer the lombok ones?